packages/xpath/vite.config.ts

```diff
@@ -125,7 +125,7 @@ export default defineConfig(({ mode }) => {
 
 			environment: TEST_ENVIRONMENT,
 			globals: false,
-			include: ['src/**/*.test.ts', 'test/native/index.ts', 'test/xforms/index.ts'],
+			include: ['test/**/*.test.ts', 'test/native/index.ts', 'test/xforms/index.ts'],
```

Comment:
😁 some tests were not even running

Reply:
Yikes! Good catch.

I think in the future, whenever I'm poking around here again, I'll eliminate the `index.ts` indirections and make this more uniform with other projects. Those were helpful for quick iteration when I was deeply focused on the `xpath` package, but I don't think they're worth the minor test run time saved now.